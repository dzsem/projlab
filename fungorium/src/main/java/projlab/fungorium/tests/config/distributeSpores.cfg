# Distribute Spores
# ------------------------------------------
# Init state
# ------------------------------------------
# 0
ADD TECTON
# 1
ADD MUSHROOMBODY 0 0
# 2
ADD TECTON
REGISTER NEIGHBOUR 0 2