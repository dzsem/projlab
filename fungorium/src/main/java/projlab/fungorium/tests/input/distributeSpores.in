# Distribute Spores
# ------------------------------------------
# Input
# ------------------------------------------
LOAD config/distributeSpores.cfg
MUSHROOMBODY 1 DISTRIBUTESPORES
# Print states
# ------------------------------------------
SAVE distributeSpores.out