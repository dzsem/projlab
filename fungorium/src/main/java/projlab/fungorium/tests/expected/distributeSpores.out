TECTON 0 1 1 0 1
TECTON 2 1 -1 0 1
MUSHROOMBODY 1 0 4 0 NORMAL 0